SR2019/2018-10-10-place-confirmation.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019
+subject: SR2019 Registration and Kickstart Announcement
```

Comment:
The subject could be a bit more direct, something like "SR2019 place confirmation and Kickstart Announcement"

Reply:
also doesn't this fail the .spelling check because it's Kickstart with a capital K?

Reply:
No. the `.spelling` syntax is such that if a word is lowercase, then the case doesn't matter, else it must conform to that case. Also, don't think the metadata is checked